### PR TITLE
Add A New Text File

### PR DESCRIPTION
**Pull request by andrewson97**
Added a new file. Please, review and approve.

### CODE REVIEW

File: andrew.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is andrew's file
```
Comment (LahiruKumaraHewagama) [inline andrew.txt]: Started with capital letters.

Comment (kmchmk) [inline andrew.txt]: Please use proper English.

Comment (andrewson97) [reply andrew.txt]: Updated. Please check.

Comment (andrewson97) [reply andrew.txt]: Updated. 
Please review.

File: andrew.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is andrew's file.
```
Comment (sarankan-24) [inline andrew.txt]: LGTM

**Review by sarankan-24 — APPROVED** [review]
LGTM

File: andrew.txt (hunk 2 above)
Comment (ghost) [inline andrew.txt]:
```suggestion
This is Andrew's file.
```

**Review by ghost — CHANGES_REQUESTED** [review]
Address all review comments.

**Comment by andrewson97** [comment]
> Address all review comments.

Updated. Please review.

**Pull request closed without merging**